package.json

```diff
@@ -5,16 +5,40 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "lint": "eslint --fix ./pages/**/*.ts[x]"
```

Comment:
--fix will autofix.

We do not want to auto fix in CircleCI. It will make the error.

Reply:
ok